client/gpg_key.go

```diff
@@ -0,0 +1,54 @@
+package client
+
+type GpgKey struct {
+	Id      string `json:"id"`
+	Name    string `json:"name"`
+	KeyId   string `json:"keyId"`
+	Content string `json:"content"`
```

Comment:
if it requires more updates on the data, please do

Reply:
Hi @sagilaufer1992 ,

In general, it's not required. What would the code do with these extra fields?

Reply:
the code wouldn't, but the user may need it, as it DOES return in the API. it may be used as output for example, or cross referenced with data from another resource to verify things in the customers data integrity (same user should have created this gpg key, and provider, for example)

Reply:
sure.
I added these two fields.